AK/UFixedBigIntDivision.h

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023, Dan Klishch <danilklishch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Diagnostics.h>
+#include <AK/UFixedBigInt.h>
+
+namespace AK {
+
+template<size_t divident_bit_size, size_t divisor_bit_size, bool restore_remainder>
+constexpr void div_mod_internal(
```

Comment:
Why is this not a standard feature anymore?

Reply:
The main argument is that this code is only used in one place in the codebase.
However, it might be just an unnecessary premature optimization of compilation time. I can merge it if you want.

Reply:
Ah I skipped over the loop setting `divisor_len` to the actually used size